paddlenlp/datasets/cote.py

```diff
@@ -68,31 +98,35 @@ def _read(self, filename, split):
             self.name]['splits'][split]
         with open(filename, 'r', encoding='utf-8') as f:
             for idx, line in enumerate(f):
-                if idx < num_discard_samples:
-                    continue
-                line_stripped = line.strip().split('\t')
-                if not line_stripped:
-                    continue
-                example = [line_stripped[indice] for indice in field_indices]
-                if split == "test":
-                    yield {"tokens": list(example[0])}
-                else:
-                    try:
-                        entity, text = example[0], example[1]
-                        start_idx = text.index(entity)
-                    except:
-                        # drop the dirty data
+                try:
+                    if idx < num_discard_samples:
                         continue
+                    line_stripped = line.strip().split('\t')
+                    if not line_stripped:
+                        continue
+                    example = [line_stripped[indice] for indice in field_indices]
+                    if split == "test":
+                        yield {"tokens": list(example[0])}
+                    else:
+                        try:
+                            entity, text = example[0], example[1]
+                            start_idx = text.index(entity)
+                        except:
+                            # drop the dirty data
+                            continue
 
-                    labels = ['O'] * len(text)
-                    labels[start_idx] = "B"
-                    for idx in range(start_idx + 1, start_idx + len(entity)):
-                        labels[idx] = "I"
-                    yield {
-                        "tokens": list(text),
-                        "labels": labels,
-                        "entity": entity
-                    }
+                        labels = ['O'] * len(text)
+                        labels[start_idx] = "B"
+                        for idx in range(start_idx + 1, start_idx + len(entity)):
+                            labels[idx] = "I"
+                        yield {
```

Comment:
建议再增加一个qid字段

Reply:
qid 是啥，行号？

Reply:
It is ok not to add qid here because the dataset dont have original unique id. Competitor could simply use incremental index as unique id when submit their results.